chart/values.yaml

```diff
@@ -545,8 +545,11 @@ config:
     worker_container_repository: '{{ .Values.images.airflow.repository | default .Values.defaultAirflowRepository }}'
     worker_container_tag: '{{ .Values.images.airflow.tag | default .Values.defaultAirflowTag }}'
     delete_worker_pods: 'True'
+    multi_namespace_mode: '{{ .Values.multiNamespaceMode }}'
 # yamllint enable rule:line-length
 
+multiNamespaceMode: 'False'
```

Comment:
Should we also update README.md? The minimum is updating the reference documentation, but I'd be happy if you would describe more about what "Multi namespace mode" means. 

https://github.com/apache/airflow/pull/10664